qiskit/transpiler/passes/routing/star_prerouting.py

```diff
@@ -331,7 +339,15 @@ def star_preroute(self, dag, blocks, processing_order):
         }
 
         def tie_breaker_key(node):
-            return processing_order_index_map.get(node, node.sort_key)
+            processing_order = processing_order_index_map.get(node, None)
+            if processing_order is not None:
+                return processing_order
+            else:
```

Comment:
This `else` block would be redundant, I suppose

Reply:
Simplified in: https://github.com/Qiskit/qiskit/pull/13736/commits/55b0ddfc1f94e51cb36fef228f025fdba12b5c26